controller/workitem.go

```diff
@@ -316,7 +317,13 @@ func (c *WorkitemController) Create(ctx *app.CreateWorkitemContext) error {
 		Fields: make(map[string]interface{}),
 	}
 	return application.Transactional(c.db, func(appl application.Application) error {
-		err := ConvertJSONAPIToWorkItem(ctx, appl, *ctx.Payload.Data, &wi, ctx.SpaceID)
+		//verify spaceID:
+		// To be removed once we have endpoint like - /api/space/{spaceID}/workitems
+		_, spaceLoadErr := appl.Spaces().Load(ctx, ctx.SpaceID)
```

Comment:
@hectorj2f done in https://github.com/fabric8-services/fabric8-wit/pull/1466/commits/6b4fd726f74f3064789d8b7c79c69701f67e21b6
Thanks, can you please approve :smile: